src/main/kotlin/no/nav/familie/ks/sak/kjerne/endretutbetaling/EndretUtbetalingAndelService.kt

```diff
@@ -138,6 +144,40 @@ class EndretUtbetalingAndelService(
             it.copy(id = 0, behandlingId = behandling.id, erEksplisittAvslagPåSøknad = false)
         endretUtbetalingAndelRepository.save(kopiertOverEndretUtbetalingAndel)
     }
+
+    fun sanityBegrunnelseTilRestFormat(
+        sanityBegrunnelser: List<SanityBegrunnelse>,
+        begrunnelse: IBegrunnelse,
+    ): List<SanityBegrunnelseMedEndringsårsakResponseDto> {
+        val sanityBegrunnelse = begrunnelse.tilSanityBegrunnelse(sanityBegrunnelser)
+
+        if (sanityBegrunnelse != null) {
+            val visningsnavn = sanityBegrunnelse.navnISystem
+
+            return listOf(
+                SanityBegrunnelseMedEndringsårsakResponseDto(
+                    id = begrunnelse,
+                    navn = visningsnavn,
+                    endringsårsaker = sanityBegrunnelse.endringsårsaker,
+                ),
+            )
+        }
+
+        return emptyList()
+    }
```

Comment:
```suggestion
    fun sanityBegrunnelseTilRestFormat(
        sanityBegrunnelser: List<SanityBegrunnelse>,
        begrunnelse: IBegrunnelse,
    ): List<SanityBegrunnelseMedEndringsårsakResponseDto> {
    fun sanityBegrunnelseTilRestFormat(
        sanityBegrunnelser: List<SanityBegrunnelse>,
        begrunnelse: IBegrunnelse,
    ): List<SanityBegrunnelseMedEndringsårsakResponseDto> {
        val sanityBegrunnelse = begrunnelse.tilSanityBegrunnelse(sanityBegrunnelser) ?: return emptyList()

        val visningsnavn = sanityBegrunnelse.navnISystem

        return listOf(
            SanityBegrunnelseMedEndringsårsakResponseDto(
                id = begrunnelse,
                navn = visningsnavn,
                endringsårsaker = sanityBegrunnelse.endringsårsaker,
            ),
        )
    }
```

Synes early return + elvis operator gjør koden lettere å lese her 😄

Reply:
Jepp enig! Tror suggested changes ble litt rart satt opp av Github, så jeg endrer det heller lokalt på samme måte som foreslått 😊